src/queryBuilder.ts

```diff
@@ -171,9 +172,10 @@ export function GenerateQuery<T extends object>(
 				.split("")
 				.join(", ")})`;
 
-			const primaryKeyStr = Array.isArray(primaryKeys)
+			const primaryKeyStr = options.uniqueColumn || (Array.isArray(primaryKeys)
```

Comment:
Is there a reason we can't just use the primary key value for this? It could use a rename - but functionally it appears to do exactly the same

Reply:
I think this covers a completely different use case, for example I have a table where the primary key is `id` but when upserting I want to use a different unique column  like `email`, `api_key` or something similar. 

I probably would not want to update the primary key to be `email` or `api_key` instead of `id`.  does that make sense?

Reply:
If we instead renamed the variable to something like `conflictConstraint`, that would demonstrate it's purpose better, and still allow you to do this.

Reply:
Are you suggesting to change `options.uniqueColumn` to `options. conflictConstraint` or you are thinking more like renaming the `primaryKeys` in the options passed to the model constructor?

Reply:
If it's the later then do you not think that would cause issues with the migrations? I was under the impression that you can also create the table using the model configurations.

Reply:
I do mean the latter. This would ideally:
- Rename the variable in the Query builder
- Edit the Model#Upsert method to have a new optional parameter for constraint columns, which will default to the primary key(s) if not already present